deemphasize units in tokens (#1065)
Heegu-sama/Homm3BG: sections/card_effect_keywords.tex

      \imagecaption{Weakness Token}\\
      \vspace{1em}
    \end{wrapfigure}
    When an effect places a Weakness Token on a unit, it specifies which side to use: $-1$ or $-2$.
    A unit with this Token suffers $-1$~\svg{attack} or $-2$~\svg{attack}, as indicated by the Token's side.
    Each unit can have only one Weakness Token at a time.
    If a unit would gain a second one, the player controlling it chooses which one to keep.

>     \textit{For example, this Token can be given to units by \wikilink{units/sorceresses}{Sorceresses} unit's ability.}

Reviewer: ```suggestion
    \textit{For example, this Token can be given to units by the \wikilink{units/sorceresses}{Sorceresses} unit's ability.}
```

      \imagecaption{Attack Token}\\
      \vspace{1em}
    \end{wrapfigure}
    When an effect places an Attack Token on a unit, it specifies which side to use: $1$ or $2$.
    Units with such a Token gain an additional 1~\svg{attack} or 2~\svg{attack}, as indicated by the Token's side.
    Each unit can have only one Attack Token at a time.
    If a unit would gain a second one, the player controlling it chooses which one to keep.

>     \textit{For example, this Token can be given to units by \wikilink{units/ogres}{Ogres} unit's ability.}

Reviewer: ```suggestion
    \textit{For example, this Token can be given to units by the \wikilink{units/ogres}{Ogres} unit's ability.}
```

      \imagecaption{Corrosion Token}\\
      \vspace{1em}
    \end{wrapfigure}
    The Corrosion Token reduces the unit's \svg{defense} by 1 to a minimum of 0.
    A unit can have only one Corrosion Token at a time.
    If a unit would gain a second Corrosion Token, discard one of them.
    Unless removed by Spells or other effects, a Corrosion Token remains on the unit until the end of Combat.

>     \textit{For example, this Token is given to enemy units by \wikilink{units/behemoths}{Behemoths}'s special ability}.

Reviewer: ```suggestion
    \textit{For example, this Token is given to enemy units by the \wikilink{units/behemoths}{Behemoths} unit's ability.}
```
Let's keep it consistent.
